configuration/cdap-site.xml

```diff
@@ -646,127 +582,88 @@
+    <name>dataset.service.output.dir</name>
+    <value>/datasets</value>
+    <description>Directory where all dataset modules archives are stored</description>
   </property>
 
-  <!-- AppFabric related changes -->
-
   <property>
-    <name>app.artifact.dir</name>
-    <value>/opt/cdap/master/artifacts</value>
-    <description>
-      Semicolon-separated list of local directories scanned for system artifacts
-      to add to the artifact repository
-    </description>
+    <name>dataset.table.prefix</name>
+    <value>${root.namespace}</value>
+    <description>Prefix for dataset table name</description>
   </property>
 
-  <property>
-    <name>app.output.dir</name>
-    <value>/programs</value>
-    <description>Directory where all archives are stored</description>
-  </property>
+  <!-- Explore Service Configuration -->
 
   <property>
-    <name>app.program.runtime.extensions.dir</name>
-    <value>/opt/cdap/master/ext/runtimes</value>
+    <name>explore.active.operation.timeout.secs</name>
+    <value>86400</value>
     <description>
-      Semicolon-separated list of local directories that are scanned for program runtime extensions
+      Timeout value in seconds for an SQL operation whose result was not
+      fetched completely
     </description>
   </property>
 
   <property>
-    <name>app.program.spark.yarn.client.rewrite.enabled</name>
-    <value>true</value>
+    <name>explore.cleanup.job.schedule.secs</name>
+    <value>60</value>
     <description>
-      Specify whether to rewrite the yarn.Client class in Spark to work
-      around the issue SPARK-13441 in some clusters.
+      Time in seconds to schedule clean-up job to timeout operations
     </description>
   </property>
 
   <property>
-    <name>app.temp.dir</name>
-    <value>/tmp</value>
-  </property>
-
-  <property>
-    <name>dataset.table.prefix</name>
-    <value>${root.namespace}</value>
-    <description>Prefix for dataset table name</description>
-  </property>
-
-  <property>
-    <name>dataset.service.output.dir</name>
-    <value>/datasets</value>
-    <description>Directory where all dataset modules archives are stored</description>
-  </property>
-
-  <property>
-    <name>dataset.executor.container.memory.mb</name>
-    <value>512</value>
+    <name>explore.enabled</name>
+    <value>true</value>
     <description>
-      Size of memory in megabytes for each dataset executor instance
+      Determines if the CDAP Explore Service (ad-hoc SQL queries) is enabled
     </description>
-    <display-name>Dataset Executor YARN Memory</display-name>
+    <display-name>Explore Enabled</display-name>
     <value-attributes>
-      <type>int</type>
-      <minimum>0</minimum>
-      <maximum>268435456</maximum>
-      <unit>MB</unit>
-      <increment-step>256</increment-step>
+      <type>boolean</type>
       <overridable>false</overridable>
     </value-attributes>
   </property>
 
```

Comment:
Correct. They're being deprecated/removed, but most importantly, changing it breaks Explore, and I want to protect users from harming themselves (where possible and easy), so I removed it in #73